torch/csrc/jit/backends/backend.h

```diff
@@ -1,13 +1,7 @@
 #pragma once
 
-#include <ATen/core/builtin_function.h>
-#include <ATen/core/stack.h>
 #include <torch/csrc/jit/backends/backend_detail.h>
 #include <torch/csrc/jit/backends/backend_interface.h>
-#include <torch/csrc/jit/backends/backend_resolver.h>
-#include <torch/csrc/jit/frontend/code_template.h>
-#include <torch/csrc/jit/frontend/resolver.h>
-#include <torch/csrc/jit/frontend/sugared_value.h>
 #include <torch/csrc/jit/python/pybind_utils.h>
```

Comment:
I don't think we can use this here, otherwise this header is not includable without libtorch_python, right?